src/index.scss

```diff
@@ -141,11 +141,13 @@ body {
     &__regulatory-icons {
         display: flex;
         align-items: center;
+        max-width: 400px;
```

Comment:
this can be `width: fit-content`

Reply:
But it doesn't work very well with tablets (less than 800px)